sdk/core/azure-core-tracing-opencensus/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 # Release History
+## Version 1.0.0-beta.5 (2019-11-25)
+For details on the Azure SDK for Java (December 2019 Preview) release refer to the [release announcement](https://aka.ms/azure-sdk-preview6-java).
+Version 1.0.0-beta.5 extended support for 0.24.0 opencensus-api package version.
```

Comment:
@kaerm / @hemantkr Could you confirm if this is the correct release announcement link?

Reply:
@adrianhall to confirm the link

Reply:
That's the wrong link, but we can correct when we get there.

Reply:
I can not confirm this this link. @adrianhall  or @kaerm   Please confirm this link.

Reply:
@hemanttanwar/@samvaity I created a link for you. It works the way it is included here. In the future you can set it up yourself to whatever you want the name to be and point to the most recent set of release notes and add adpteam as a co-owner.